mobile/lib/features/trade/trade_screen.dart

```diff
@@ -286,7 +286,7 @@ class TradeScreen extends StatelessWidget {
                         tradeBottomSheet(context: context, direction: Direction.long);
                       }
                     },
-                    label: const Text("Buy"),
+                    label: const Text("Buy / Long"),
```

Comment:
Obviously subjective, but I like the "Buy" and "Sell" labels better than "Buy / Long" and "Sell / Short".

Reply:
sure, I can remove that change from this changeset! 
It was one of the feedback from the conference - people are *really* confused by only 'buy' and 'sell' (and asked questions like "can I *buy* Bitcoin here? what it is this?".
I took the new naming after Kollider (which has it like that), and thought that it is only an improvement - if less people get confused, but still understandable for pros. (especially that we're targetting *retail* traders).